srcpkgs/rclone/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'rclone'
 pkgname=rclone
 version=1.54.0
-revision=1
+revision=2
 wrksrc="rclone-v${version}"
 build_style=go
 go_import_path=github.com/rclone/rclone
-go_ldflags="-extldflags=-fuse-ld=bfd"
+go_ldflags="-extldflags=-fuse-ld=bfd -X github.com/rclone/rclone/fs.Version=v${version}"
 short_desc="Rsync for cloud storage"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="MIT"
 homepage="https://rclone.org/"
-changelog="https://raw.githubusercontent.com/rclone/rclone/master/docs/content/changelog.md"
```

Comment:
I think we should keep this one... I believe the general preference is for raw text changelogs, when available.